packages/web-forms/package.json

```diff
@@ -46,25 +46,26 @@
     "test:types:node": "vue-tsc --project ./tsconfig.node.json --emitDeclarationOnly false --noEmit",
     "test:types:vitest": "vue-tsc --project ./tsconfig.vitest.json --emitDeclarationOnly false --noEmit"
   },
-  "dependencies": {
-    "@getodk/xforms-engine": "0.1.0",
-    "primevue": "^3.49.1",
-    "primeflex": "^3.3.1",
-    "vue": "^3.4.19",
-    "vue-router": "^4.3.0"
-  },
   "devDependencies": {
-    "@faker-js/faker": "8.4.1",
-    "@playwright/test": "^1.42.1",
-    "@types/ramda": "0.29.11",
+    "@faker-js/faker": "^8.4.1",
+    "@getodk/xforms-engine": "0.1.1",
+    "@playwright/test": "^1.44.1",
+    "@types/ramda": "^0.30.0",
     "@vitejs/plugin-vue": "^5.0.4",
     "@vitejs/plugin-vue-jsx": "^3.1.0",
-    "@vue/test-utils": "^2.4.4",
+    "@vue/test-utils": "^2.4.6",
     "jsdom": "^24.0.0",
+    "primeflex": "^3.3.1",
+    "primevue": "^3.52.0",
     "primevue-sass-theme": "https://github.com/primefaces/primevue-sass-theme.git#3.49.1",
-    "ramda": "0.29.1",
-    "sass": "^1.71.1",
-    "vite": "^5.1.5",
-    "vitest": "^1.3.1"
+    "ramda": "^0.30.0",
+    "sass": "^1.77.2",
+    "vite": "^5.2.11",
+    "vitest": "^1.6.0",
+    "vue": "^3.4.27",
+    "vue-router": "^4.3.2"
+  },
+  "peerDependencies": {
+    "vue": "^3.4.27"
```

Comment:
Will it block Central integration if we don't allow an older version than we've pinned? If so, I'd be happy to pick a minimum that works for both. Looks like it's currently locked at 3.3.4, would that work?

I'd prefer to set the minimum to our installed version, whatever that turns out to be. If there are downstream issues with a lower version, it can be a real pain to investigate.

`^` is **more lenient** for future versions than `~`, so I think we should still use that when we agree on an appropriate minimum.

Reply:
I've downgraded to 3.3.4 (pinned exact in `devDependencies`, and `^` in `peerDependencies`) in hopes this will be satisfactory. If it is, and if everything else looks good, please feel free to merge & release at your nearest convenience. You do not need to wait for me to get a last look, as this PR is blocking all other work.

Note: I don't recall why `vue` had to be installed both at the monorepo root and in the `web-forms` package, but I do recall that it had to be to make all the tooling play nice together (or as nicely as it does play together, at any rate). Anyway, we'll need to remember to keep both versions in sync whenever we update this special case. Before I caught that, I was getting a ton of mysterious hard-to-search errors when I tried to run tests. I see that Central's update to 3.3.4 was 10 months ago, hopefully we can update both before our next release.